src/guide/10-performance-troubleshooting/1-perf.rst

```diff
@@ -1,2 +1,17 @@
-Perf
-===================
+Performance Troubleshooting
```

Comment:
Release build, Seatbelts off. The performance of any other version can be assumed bad.

Reply:
Related: https://github.com/FLAMEGPU/FLAMEGPU2/issues/526